.gitignore

```diff
@@ -1,3 +1,7 @@
+
+# this is a test
+
+
```

Comment:
The hard part of programming is deciding how you'll boil the messy real world down to the numbers and words you can store in a database. ... 

Often once you have a good data model, the code writes itself. ...

When we solve a problem for a client, we will usually end up touching every level: UI, front end logic, back end logic, and database. But getting it right in the database is where it starts. "What's the right thing to do with the data?"

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix